lib/formulary/qhp_importer.rb

```diff
@@ -19,8 +19,10 @@ def import
         puts "Reading #{filenames.length} file(s) from #{url}"
 
         filenames.each do |filename|
-          qhp_raw_data = File.read(filename)
-          qhp_json.concat(JSON.parse(qhp_raw_data, symbolize_names: true))
+          qhp_raw_data = File.read(filename).split("\n")
```

Comment:
The files are small enough where reading into memory isn't a problem yet.  We may want to keep eyes on the memory footprint of the job executor in the CircleCI job